rel-eng/comps/comps-katello-server-rhel6.xml

```diff
@@ -11,12 +11,18 @@
     <description>Katello Server Packages</description>
     <uservisible>true</uservisible>
     <packagelist>
+       <!-- elasticsearch packages -->
        <packagereq type="default">elasticsearch</packagereq>
-       <packagereq type="default">ruby193-rubygem-katello</packagereq>
+       <packagereq type="default">ruby193-rubygem-tire</packagereq>
+       <packagereq type="default">lucene4</packagereq>
+       <packagereq type="default">lucene4-contrib</packagereq>
+       <packagereq type="default">sigar-java</packagereq>
+       <packagereq type="default">sigar</packagereq>
+       <packagereq type="default">snappy-java</packagereq>
+
+       <!-- katello core packages -->
        <packagereq type="default">katello</packagereq>
-       <packagereq type="default">katello-agent</packagereq>
```

Comment:
Also keep in mind that the capulse will need katello-agent, today they can just install this one rpm.  With this change they can't (have to configure the client repo directly).